test/integ/test_key_pair_authentication.py

```diff
@@ -14,6 +17,46 @@
 import snowflake.connector
 
 
+@pytest.mark.skipolddriver
+def test_get_token_from_private_key():
+    test_user = "python_test_keypair_user_" + str(uuid.uuid4()).replace("-", "_")
+    current_dir = path.dirname(path.realpath(__file__))
+    private_key_file_path = path.join(
+        current_dir, "../data/rsa_keys", "rsa_key_encrypted.p8"
```

Comment:
The separator embedded like that won't work on Windows
```suggestion
        current_dir, "..", "data", "rsa_keys", "rsa_key_encrypted.p8"
```